grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowConfigDrawer.tsx

```diff
@@ -0,0 +1,442 @@
+import React, { useEffect, useState } from 'react';
+
+import { css } from '@emotion/css';
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import {
+  Drawer,
+  Field,
+  HorizontalGroup,
+  Input,
+  VerticalGroup,
+  Icon,
+  useStyles2,
+  Button,
+  LoadingPlaceholder,
+  Select,
+  SelectBaseProps,
+} from '@grafana/ui';
+import { observer } from 'mobx-react';
+import { Controller, useForm } from 'react-hook-form';
+
+import { IntegrationInputField } from 'components/IntegrationInputField/IntegrationInputField';
+import { RenderConditionally } from 'components/RenderConditionally/RenderConditionally';
+import { Text } from 'components/Text/Text';
+import { ActionKey } from 'models/loader/action-keys';
+import { ApiSchemas } from 'network/oncall-api/api.types';
+import { useCurrentIntegration } from 'pages/integration/OutgoingTab/OutgoingTab.hooks';
+import { useStore } from 'state/useStore';
+import { URL_REGEX } from 'utils/consts';
+import { useIsLoading } from 'utils/hooks';
+import { OmitReadonlyMembers } from 'utils/types';
+import { openNotification } from 'utils/utils';
+
+interface ServiceNowConfigurationDrawerProps {
+  onHide(): void;
+}
+
+enum OnCallAGStatus {
```

Comment:
worth to check if it's not already defined in autogenerated types. Then you could use it from there instead of creating new enum

Reply:
Such statuses dont' show up in there in this form, we have this instead

```jsx
status?: (0 | 1 | 2 | 3)[];
```